README.md

```diff
@@ -37,6 +37,7 @@ yarn add fnlint
 - **`files`** Glob string for files to lint relative to base path.
 - **`format`** File name format.
 - **`reporter`** (optional) Set to false to turn off console reporter.
+- **`directories`** (optional) Set to true to lint full path including directories.
```

Comment:
Why do you want to give the option to either lint files or directories? Isn't there a way for users to give paths and we figure out if it is file or directory? In case of file we lint file, in case of directory we, recursively lint. What do you think?